__mocks__/handlers/self.handler.ts

```diff
@@ -0,0 +1,40 @@
+import { rest } from 'msw';
+const URL = process.env.NEXT_PUBLIC_BASE_URL;
+
+export const userSelfData = {
```

Comment:
please take a pull from develop, there should already be files having mock users data, creating a new one is not required IMO